lib/active_shipping/carriers/usps.rb

```diff
@@ -168,15 +168,15 @@ class USPS < Carrier
       /Delivery status information is not available/
     ]
 
-    ESCAPING_AND_SYMBOLS = /&amp;lt;\S*&amp;gt;/
-    LEADING_USPS = /^USPS/
+    ESCAPING_AND_SYMBOLS = /&lt;\S*&gt;/
+    LEADING_USPS = /^USPS /
```

Comment:
These changes were needed due to small parsing differences between Nokogiri and REXML.